### PR TITLE
Add ernie seqeuence classification doc

### PR DESCRIPTION
**Pull request by joey12300**
### PR types

Others
### PR changes

Models
### Description

Add ernie seqeuence classification doc

### CODE REVIEW

File: examples/experimental/faster_ernie/seq_cls/train.py [hunk 1]
```diff
@@ -165,12 +159,24 @@ def do_train():
                 optimizer.step()
             lr_scheduler.step()
             optimizer.clear_grad()
+
+            global_step += 1
+            if global_step % 10 == 0:
```
Comment (wawltor) [inline examples/experimental/faster_ernie/seq_cls/train.py]: 这里加个logging_steps 和下面的文本分类的日志打印方式对齐

File: examples/experimental/faster_ernie/seq_cls/train.py [hunk 2]
```diff
@@ -165,12 +159,24 @@ def do_train():
...
+                time_diff = time.time() - tic_train
+                total_train_time += time_diff
+                print(
+                    "global step %d, epoch: %d, batch: %d, loss: %.5f, accu: %.5f, speed: %.2f step/s"
```
Comment (wawltor) [inline examples/experimental/faster_ernie/seq_cls/train.py]: accu->acc或者accuracy

File: examples/text_classification/pretrained_models/train.py [hunk 3]
```diff
@@ -204,15 +198,28 @@ def do_train():
             lr_scheduler.step()
             optimizer.clear_grad()
 
+            global_step += 1
+            if global_step % args.logging_steps == 0 and rank == 0:
+                time_diff = time.time() - tic_train
+                total_train_time += time_diff
+                print(
+                    "global step %d, epoch: %d, batch: %d, loss: %.5f, accu: %.5f, speed: %.2f step/s"
```
Comment (wawltor) [inline examples/text_classification/pretrained_models/train.py]: accu同上

File: examples/text_classification/pretrained_models/train.py [hunk 4]
```diff
@@ -204,15 +198,28 @@ def do_train():
...
+                    % (global_step, epoch, step, loss, acc, 10 / (time_diff)))
```
Comment (wawltor) [inline examples/text_classification/pretrained_models/train.py]: 这里的10改成logging_steps?

File: examples/experimental/faster_ernie/seq_cls/train.py (hunk 1 above)
Comment (joey12300) [reply examples/experimental/faster_ernie/seq_cls/train.py]: done

File: examples/experimental/faster_ernie/seq_cls/train.py (hunk 2 above)
Comment (joey12300) [reply examples/experimental/faster_ernie/seq_cls/train.py]: done

File: examples/text_classification/pretrained_models/train.py (hunk 3 above)
Comment (joey12300) [reply examples/text_classification/pretrained_models/train.py]: done

File: examples/text_classification/pretrained_models/train.py (hunk 4 above)
Comment (joey12300) [reply examples/text_classification/pretrained_models/train.py]: done

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**